themes/root/templates/icons/Remix.phtml

```diff
@@ -0,0 +1,3 @@
+<svg class="icon--svg"<?=$this->attrs ?> aria-hidden="true">
```

Comment:
If we go forward with including Remix icons, we should update our icons page in the wiki to give them credit, and perhaps also include a copy of their license in the directory with the icons. Might be worth adding a TODO comment above.

Reply:
I'm picking on Remix right now because they have a lot of options for import (svg, sprite, font) so I can work with two robust fonts. I do like it though and crediting them is the least we could do.

Reply:
I see that the Remix file itself contains licensing information, so I don't think we need to add a separate license file. However, where did we land on the use of Remix? I see that right now, the configuration is actually using Remix for a couple of things. Was this only for testing/demo purposes, or is this a real change that we want to merge?